ext/sockets/sockets.stub.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/** @return int|false */
+function socket_select(array $read_fds, array $write_fds, array $except_fds, $tv_sec, int $tv_usec = 0) {}
```

Comment:
Missing param type for `$tv_sec`?

Reply:
Was a bit unsure about this one since it is specified as `z!`. Should I update the declaration to `?int`?

Reply:
I think `?int` is correct here. We should probably change ZPP to except `l?` in the first place.

Reply:
I have updated the commit to do the changes, I'm not sure if it is appropriate or not to do it within this PR. If it is not I can remove it and open a dedicated PR.

Reply:
Fine for this PR. Thanks!